solvebio/resource/datasetcommit.py

```diff
@@ -20,11 +21,21 @@ class DatasetCommit(CreateableAPIResource, ListableAPIResource,
         ('created_at', 'Created'),
     )
 
+    @property
+    def dataset_id(self):
+        return self['dataset']['id']
```

Comment:
I'm surprised there isn't a dataset_id attribute on this object... or is it write only?

Reply:
hmm... ya weird. that is there already. removing